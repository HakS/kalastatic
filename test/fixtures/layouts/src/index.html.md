---
title: Jade
layout: _layout.jade
---
This is an article.
