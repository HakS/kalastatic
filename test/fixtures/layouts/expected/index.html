<!DOCTYPE html>
<html lang="en">
  <head>
    <title>Jade</title>
  </head>
  <body><p>This is an article.</p>
</body>
</html>